src/components/ha-markdown.ts

```diff
@@ -16,6 +16,9 @@ class HaMarkdown extends UpdatingElement {
   }
 
   private async _render() {
+    this.style.display = "block";
+    this.style.padding = "0 16px 16px";
```

Comment:
Do we always want this padding? 

Also you should not set it on the element itself, it will overrule styles set by its parents.

We should use `:host() {}` (but that requires shadowRoot)

I see this element doesn't have a `shadowRoot`, we should add that if we are going to add styles, otherwise the styles will not be contained within this element.